contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -3930,7 +3937,7 @@ load_categories_hash(RawStmt *cats_sql, MemoryContext per_query_ctx)
 		elog(ERROR, "load_categories_hash: SPI_connect returned %d", ret);
 
 	/* Retrieve the category name rows */
-	ret = SPI_execute_raw_parsetree(cats_sql, true, 0);
+	ret = SPI_execute_raw_parsetree(cats_sql, sourcetext, true, 0);
 	proc = SPI_processed;
```

Comment:
It'll better to change this var name to tuple_processed.

Proc has some other meaning in the code and may cause misunderstanding.

Reply:
yes, will change the var name